helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -99,6 +99,18 @@ splunkPlatform:
     #   requests_per_second is the average number of requests per seconds.
     queueSize: 5000
 
+  # When enabled, it uses file_storage extension to persist the queue data.
+  # This can be used to prevent data-loss during restart
+  # NOTE: The File Storage extension will persist state to the node's local file system.
+  # While using the persistent queue it is advised to increase memory limit for agent (agent.resources.limits.memory)
+  # to 1Gi.
+  persistentQueueEnabled:
```

Comment:
Does it make more sense to nest this in `sendingQueue` since that's what uses it?